Description: install data file to meet the need of multi-arch
 Install data files to ${CMAKE_INSTALL_LIBDIR}
Author: Aron Xu <aron@debian.org>
Index: opencc/CMakeLists.txt
===================================================================
--- opencc.orig/CMakeLists.txt
+++ opencc/CMakeLists.txt
@@ -69,7 +69,7 @@ set (DIR_PREFIX ${CMAKE_INSTALL_PREFIX})
 set (DIR_INCLUDE ${DIR_PREFIX}/include/)
 set (DIR_SHARE ${DIR_PREFIX}/share/)
 set (DIR_ETC ${DIR_PREFIX}/etc/)
-set (DIR_LIBRARY ${DIR_PREFIX}/lib/)
+set (DIR_LIBRARY ${CMAKE_INSTALL_LIBDIR})
 
 if (DEFINED SHARE_INSTALL_PREFIX)
   set (DIR_SHARE ${SHARE_INSTALL_PREFIX})
Index: opencc/src/CMakeLists.txt
===================================================================
--- opencc.orig/src/CMakeLists.txt
+++ opencc/src/CMakeLists.txt
@@ -83,8 +83,8 @@ endif (CMAKE_BUILD_TYPE MATCHES Debug)
 install(
   TARGETS
     libopencc
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib/static
+  LIBRARY DESTINATION ${DIR_LIBRARY}
+  ARCHIVE DESTINATION ${DIR_LIBRARY}
 )
 
 install(
